### ISSUE TITLE
window.caligon.status4evar is undefined 

### ISSUE THREAD

**Issue by drankinatty** [SparkyBluefang/S4E#29]
Matt, great job with the addon. I have an issue that stated on upgrade from FF35 to FF36 (I hate upgrades). When FF starts, or after a crash, none of the previous tabs or homepage are restored. Starting from the command line, the following stood out:

console.error:
  [CustomizableUI]
  TypeError: window.caligon.status4evar is undefined -- resource://status4evar/Australis.jsm:186
WARNING: content window passed to PrivateBrowsingUtils.isWindowPrivate. Use isContentWindowPrivate instead (but only for frame scripts). I don't know if something in FF 35->36 changed that effects s4e, but it was very apparent, the FF35->36 was the cause of the homepage no longer being shown and the crash reporter window not to appear after a crash.

The 'window.caligon.status4evar is undefined' error followed the only other error shown on startup:

1428021849673   addons.xpi      WARN    Exception running bootstrap method startup on {d10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d}: [Exception... "Component returned failure code: 0xc1f30100 (NS_ERROR_FACTORY_EXISTS) [nsIComponentRegistrar.registerFactory]"  nsresult: "0xc1f30100 (NS_ERROR_FACTORY_EXISTS)"  location: "JS frame :: resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///home/david/.mozilla/firefox/nbveu8do.default/extensions/%7Bd10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d%7D.xpi!/bootstrap.js -> jar:file:///home/david/.mozilla/firefox/nbveu8do.default/extensions/%7Bd10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d%7D.xpi!/lib/main.js :: registerPublicAPI :: line 56"  data: no] Stack trace: registerPublicAPI()@resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///home/david/.mozilla/firefox/nbveu8do.default/extensions/%7Bd10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d%7D.xpi!/bootstrap.js -> jar:file:///home/david/.mozilla/firefox/nbveu8do.default/extensions/%7Bd10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d%7D.xpi!/lib/main.js:56 < resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///home/david/.mozilla/firefox/nbveu8do.default/extensions/%7Bd10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d%7D.xpi!/bootstrap.js -> jar:file:///home/david/.mozilla/firefox/nbveu8do.default/extensions/%7Bd10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d%7D.xpi!/lib/main.js:25 < require()@resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///home/david/.mozilla/firefox/nbveu8do.default/extensions/%7Bd10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d%7D.xpi!/bootstrap.js:137 < startup()@resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///home/david/.mozilla/firefox/nbveu8do.default/extensions/%7Bd10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d%7D.xpi!/bootstrap.js:21 < XPI_callBootstrapMethod()@resource://gre/modules/addons/XPIProvider.jsm:4436 < XPI_startup()@resource://gre/modules/addons/XPIProvider.jsm:2160 < callProvider()@resource://gre/modules/AddonManager.jsm:208 < _startProvider()@resource://gre/modules/AddonManager.jsm:667 < AMI_startup()@resource://gre/modules/AddonManager.jsm:824 < AMP_startup()@resource://gre/modules/AddonManager.jsm:2402 < AMC_observe()@resource://gre/components/addonManager.js:55 < <file:unknown>

I admit, I suck at debugging FF addons. I'm not 100% sure those two errors are related. But whatever the reason, I would like to see if the issue could be fixed. I'm happy to do more testing on my end, if you will let me know what you need. I'll get it. Keep up the great work!

**Comment by alexolog** [comment]
Happens to me as well

**Comment by Zefling** [comment]
I think I have the same problem with Fx 40.0 : 
```cmd
console.error:                                                                     
  [CustomizableUI]                                                                 
  TypeError: window.caligon.status4evar is undefined -- resource://status4evar/Australis.jsm:186
console.error:                                                                     
  [CustomizableUI]                                                                 
  TypeError: window.caligon.status4evar is undefined -- resource://status4evar/Australis.jsm:186
console.error:                                                                     
  [CustomizableUI]                                                                 
  NotFoundError: Node was not found -- :1152                                       
console.error:                                                                     
  [CustomizableUI]                                                                 
  TypeError: window.caligon.status4evar is undefined -- resource://status4evar/Australis.jsm:186
console.error:                                                                     
  [CustomizableUI]
  NotFoundError: Node was not found -- :941
```
Firefox is launched without chrome (no interface, no menu, nothing).

**Comment by SparkyBluefang** [comment]
This should be fixed by 823cf1b75005c3b50a8fca68e0bcf9eb9910c475

Give the latest beta version a try.

https://addons.mozilla.org/en-US/firefox/addon/status-4-evar/versions/?page=1#version-2015.08.19.02b
